### ISSUE TITLE
[WinForms ComboBox] selection by index is not possible for UIA backend

### ISSUE THREAD

**Issue by Vasanthkvr** [pywinauto/pywinauto#473]
Hi,
I tried to select the item in combo box using index. But i get the following error

Version: pywinauto-0.6.4
Ref:
`Application(backend="uia")` using UIA as backend
`self.mainForm.child_window(auto_id='NonEditableCombo', control_type='ComboBox').select(1)`

I get the following exception:
```python
Traceback (most recent call last):
  File ".../TestCaseSampleApp.py", line 18, in <module>
    TestAllControls()
  File ".../TestCaseSampleApp.py", line 10, in TestAllControls
    objSampleApp.combobox() #issue
  File "....\SampleApp.py", line 34, in combobox
    self.mainForm.child_window(auto_id='NonEditableCombo', control_type='ComboBox').select(1)
  File "....\uia_controls.py", line 158, in select
    self.expand()
  File "...\uiawrapper.py", line 457, in expand
    self.iface_expand_collapse.Expand()
  File "...\uiawrapper.py", line 129, in __get__
    value = self.fget(obj)
  File "...\uiawrapper.py", line 207, in iface_expand_collapse
    return uia_defs.get_elem_interface(elem, "ExpandCollapse")
  File "....\uia_defines.py", line 219, in get_elem_interface
    raise NoPatternInterfaceError()
pywinauto.uia_defines.NoPatternInterfaceError
```

**Comment by Vasanthkvr** [comment]
Hi,
I have two combo box one is the editable combo box and another one is non editable combo box.
In Editable combo box support to type the value in edit box and also we can able select the item from drop down list.
In Non Editable combo box support only the value selection through drop down and select the item.
That's the difference between Editable and non Editable [ControlType.ComboBox].

Please find the following code to have issue clarity.
**Step1**: Attach the existing application [backend="uia"]
sampleApp = Application(backend="uia").connect(path='WinFormsApplication.exe')
**Step2**: Find the combo box and select item 0
combo = self.mainForm.child_window(auto_id='EditableCombo', control_type='ComboBox')
combo .select(0)
**Issue**
The above select didn't expand the combo box and select the item 0.
Exception:
Traceback (most recent call last):
  File "../TestCaseSampleApp.py", line 20, in <module>
    TestAllControls()
  File "../TestCaseSampleApp.py", line 11, in TestAllControls
    objSampleApp.comboboxEditable() #issue
  File "..\SampleApp.py", line 45, in comboboxEditable
    combo.select(1)
  File "..\uia_controls.py", line 158, in select
    self.expand()
  File "..\uiawrapper.py", line 510, in expand
    self.iface_expand_collapse.Expand()
  File "..\uiawrapper.py", line 131, in __get__
    value = self.fget(obj)
  File "..\uiawrapper.py", line 209, in iface_expand_collapse
    return uia_defs.get_elem_interface(elem, "ExpandCollapse")
  File "..\uia_defines.py", line 234, in get_elem_interface
    raise NoPatternInterfaceError()
pywinauto.uia_defines.NoPatternInterfaceError

**Comment by vasily-v-ryabov** [comment]
Well, I've found WinForms combo box sample app and this combo box doesn't support ExpandCollapsePattern, but there is ValuePattern. So this code might work for you:
```python
combo.iface_value.SetValue("The exact text of the item")
```
Selection by index is possible when you're using Win32 backend:
```python
app=Application(backend="win32").connect(path='WinFormsApplication.exe')
combo = app.mainForm.ComboBox1
combo.select(0)
```
Though `auto_id` property is not available in Win32 backend so finding the combo box is not that easy. I think we can improve this in the future (since it's a standard WinForms control). So will keep the issue open as an enhancement for usability.

**Comment by vasily-v-ryabov** [comment]
Possible workaround found by another guy: https://stackoverflow.com/q/49436614/3648361

**Comment by vasily-v-ryabov** [comment]
Hi @Vasanthkvr the issue seems already fixed in `pywinauto==0.6.6`. Please update it by `pip install -U pywinauto` and try again.
